keras/legacy/interfaces.py

```diff
@@ -133,3 +133,64 @@ def wrapper(*args, **kwargs):
 
         return func(*args, **kwargs)
     return wrapper
+
+
+def legacy_prelu_support(func):
+    """Function wrapper to convert the `PReLU` constructor from Keras 1 to 2.
+
+    # Arguments
+        func: `__init__` method of `PReLU`.
+
+    # Returns
+        A constructor conversion wrapper.
+    """
+    @six.wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 1:
+            # The first entry in `args` is `self`.
+            raise TypeError('The `PReLU` layer can have no '
```

Comment:
This layer is commonly used with a positional argument, e.g. `PReLU(0.4)`. This should still work.